README.md

```diff
@@ -1,2 +1,169 @@
-# portfolio_project
-This is almost a clone of 1st week of portfolio assignment with semantic tags of HTML -> to review by my own.
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
```

Comment:
- [ ] like @OyePriscilla has said,  Please, make sure you add the 'why' you've suggested any changes, this creates a conviction for the reviewee 😀.

As an example, this:
Kindly ensure that the list of icons is an `a` tag wrapped in a `li` tag then all wrapped in a `ul` tag.

something like this 

```
<ul>
  <li><a>home</a></li>
  <li><a>house</a></li>
  <li><a>yard</a></li>
  <li><a>apartment</a></li>
</ul>
```

**This is good for semantic and user experience reasons respectively and also enables one to connect with the icons using the `href` attribute.**

The above review shows that the phrase "This is good for..." is the "why" for the review.